iothub/service/src/Feedback/Models/FeedbackBatch.cs

```diff
@@ -21,7 +22,8 @@ public class FeedbackBatch
         /// To unit test methods that use this type as a response, inherit from this class and give it a constructor
         /// that can set the properties you want.
         /// </remarks>
-        protected internal FeedbackBatch()
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public FeedbackBatch()
```

Comment:
No parameterized constructor, so this shouldn't be reqiured, right?

Reply:
Same as above.